CHANGELOG

```diff
@@ -1,4 +1,14 @@
-Version 1.2.0 (2017-12-XX)
+Version 1.3.0 ()
```

Comment:
Is it intentional? to be filled at last minute?

Reply:
yes, I'll fill it when removing rc commit (I hope to not forget it)